CONTRIBUTING.md

```diff
@@ -79,8 +79,6 @@ We provide a script that can automatically create all the necessary files for yo
 
 - Make sure Docker is running.
```

Comment:
I think we can just remove this section and mention it under "How to run tests".

Reply:
done